static/src/javascripts/projects/common/modules/experiments/tests/video-caption.js

```diff
@@ -1,36 +1,46 @@
-define(['qwery',
-        'common/utils/config'
-], function(
+define([
+    'common/utils/config',
+    'qwery',
+    'bonzo',
+    'common/utils/$'
+], function (
+    config,
     qwery,
-    config
-
-) {
-    return function() {
+    bonzo,
+    $
+    ){
+    return function()
+    {
         this.id = 'VideoCaption';
         this.start = '2016-07-18';
-        this.expiry = '2016-07-25';
+        this.expiry = '2016-09-01';
         this.author = 'Gideon Goldberg';
-        this.showForSensitive = true;
-        this.description = 'Test if increasing the prominence of the video caption on in-article videos leads to more plays.';
-        this.audience = 1;
+        this.description = 'Increase the prominence of the video caption on in-article videos.';
+        this.audience = 0;
         this.audienceOffset = 0;
-        this.successMeasure = 'Video starts';
-        this.audienceCriteria = '';
-        this.dataLinkNames = '';
-        this.idealOutcome = '';
+        this.successMeasure = 'Video starts.';
+        this.audienceCriteria = 'Users viewing an article with a video embedded.';
+        this.dataLinkNames = 'video caption play';
+        this.idealOutcome = 'Video starts are increased';
         this.canRun = function() {
             return config.page.contentType === 'Article' &&  qwery('[data-component="main video"]').length > 0;
         };
 
         this.variants = [
             {
-                id: 'baseline1',
+                id: 'control',
+                test: function () {
+                }
+            },
+            {
+                id: 'caption-overlay',
                 test: function () {
                 }
             },
             {
-                id: 'baseline2',
+                id: 'caption-larger-text',
                 test: function () {
+                    bonzo($('.caption--main a')).addClass('caption--large');
```

Comment:
Actually looks like this modifier is targeted only at the main media https://github.com/guardian/frontend/blob/master/article/app/views/fragments/mainMedia.scala.html#L44